### ISSUE TITLE
updates

### ISSUE THREAD

**Issue by yodaphone** [ReceiptManager/receipt-parser-server#60]
hi 

i would like to use this for my home and wanted to know if you are still actively maintaning this ?

**Comment by monolidth** [comment]
Yeah, it should work fine. However, I will replace the legacy parcer with an ML parser in the future.
